vmdb/app/models/miq_request.rb

```diff
@@ -66,34 +66,8 @@ class MiqRequest < ActiveRecord::Base
     }
   }
 
-  REQUEST_TYPES = {
-    :MiqProvisionRequest             => {
-      :template          => "VM Provision",
-      :clone_to_vm       => "VM Clone",
-      :clone_to_template => "VM Publish",
-    },
-    :MiqProvisionRequestTemplate     => {
-      :template => "VM Provision Template"
-    },
-    :MiqHostProvisionRequest         => {
-      :host_pxe_install => "Host Provision"
-    },
-    :VmReconfigureRequest            => {
-      :vm_reconfigure => "VM Reconfigure"
-    },
-    :VmMigrateRequest                => {
-      :vm_migrate => "VM Migrate"
-    },
-    :AutomationRequest               => {
-      :automation => "AutomationRequest"
-    },
-    :ServiceTemplateProvisionRequest => {
-      :clone_to_service => "Service Provision"
-    },
-    :ServiceReconfigureRequest       => {
-      :service_reconfigure => "Service Reconfigure"
-    }
-  }
+  REQUEST_TYPES_BACKEND_ONLY = {:MiqProvisionRequestTemplate => {:template => "VM Provision Template"}}
```

Comment:
I wanted it to be obvious that this is intentionally added and is only for backend use